woocommerce/woocommerce-gutenberg-products-block
# Update Banner Pattern To Replace Unsplash Image

This PR replaces the Unsplash image with a CCO licensed image from Pxhere.

It also updates the CTA button to link to the Shop page, by default.





### Screenshots



| Before | After |
| ------ | ----- |
| ![CleanShot 2023-06-08 at 15 15 25](https://github.com/woocommerce/woocommerce-blocks/assets/481776/6cc2d362-24e5-4a23-8694-511bc0e62fc4) | ![CleanShot 2023-06-08 at 15 34 37](https://github.com/woocommerce/woocommerce-blocks/assets/481776/338a024b-7bf1-4963-9206-9a84ceeb9077) |

### Testing



1. From the editor (post, page, etc.) apply the `Banner` pattern.
2. Confirm the image matches the "After" version in the screenshot above in both the admin and the front end.
3. Confirm the CTA button links to the Shop page on the front end.
4. Confirm vertical spacing is fixed on the text column (extra padding on the top is removed).


### WooCommerce Visibility




### Changelog

> Update the banner pattern to replace the Unsplash image with a CCO licensed image from Pxhere.
